sensors-cms-frontend/src/pages/Dashboard.tsx

```diff
@@ -99,8 +99,8 @@ function Dashboard() {
       const jsonData = isJson(msg)
       dataRefreshCounter += 1
 
-      if (dataRefreshCounter >= dataRefreshRate) {
-        dataRefreshCounter = 0
+      if (dataRefreshCounter === dataRefreshRate) {
+        dataRefreshCounter = 1
```

Comment:
does this break or improve functionality on how frequent the data refreshes and can it be tested?